### PR TITLE
Translations for xsns_53_sml.ino (en-GB, de-DE and pt-PT)

### LINKED ISSUE #7916: German languange texts with default en-GB configuration
### PROBLEM DESCRIPTION

German texts with webUI in english:

image

This is straight from a compiled Tasmota development version, on adding the procedures and lines stated in the wiki for Smart Meters.

No `MY_LANGUAGE` settings were changed or defined.

### REQUESTED INFORMATION
- [X] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [X] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/#/help/FAQ)
- [x] Searched the problem in the [forum](https://groups.google.com/d/forum/sonoffusers)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): Wemos D1 mini
- [x] Tasmota binary firmware version number used: development as 13/03/2020
  - [ ] Pre-compiled
  - [x] Self-compiled
    - [x] IDE / Compiler used: Arduino 1.8.3
- [X] Flashing tools used: USB
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```
  Configuration output here:

Default.

```
### TO REPRODUCE
N/A

### EXPECTED BEHAVIOUR
The texts should be all in english (default en-GB)

### ADDITIONAL CONTEXT
N/A

### PR DESCRIPTION
**Pull request by tiagofreire-pt**
## Description:

**Related issue (if applicable):** fixes #7916

## Checklist:
  - [X] The pull request is done against the latest dev branch
  - [X] Only relevant files were touched
  - [X] Only one feature/fix was added per PR.
  - [x] The code change is tested and works on core Tasmota_core_stage
  - [X] The code change pass travis tests. **Your PR cannot be merged unless tests pass**
  - [X] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

### CODE REVIEW

**Comment by Jason2866** [comment]
If you do language adaptions, ALL language files have to be touched.
You can copy in all other languages files the english content.
The community will do the translation.

**Comment by tiagofreire-pt** [comment]
> 
> 
> If you do language adaptions, ALL language files have to be touched.
> Please copy the original content from german in german language file
> You can copy in all other languages files the english content.
> 
> The community will do the translation.

Done. :)

**Comment by Jason2866** [comment]
Please merge the german translation PR in your PR :-)

**Comment by tiagofreire-pt** [comment]
> 
> 
> Please merge the german translation PR in your PR :-)

Done. Tks! :)

File: tasmota/language/en-GB.h [hunk 1]
```diff
@@ -512,6 +512,46 @@
...
+#define D_Spannung_L2 "Voltage L2"
+#define D_Spannung_L3 "Voltage L3"
+#define D_METERNR "Meter_number"
+#define D_METERSID "Service ID"
+#define D_GasIN "Counter"                // Gas-Verbrauch
+#define D_H2oIN "Counter"                // H2o-Verbrauch
+#define D_StL1L2L3 "Current L1+L2+L3"
+#define D_SpL1L2L3 "Voltage L1+L2+L3/3"
+
+// xsns_53_sml.ino
```
Comment (ascillato2) [inline tasmota/language/en-GB.h]: Duplicated entries

**Review by ascillato2 — CHANGES_REQUESTED** [review]
Duplicated entries on en-GB-h file

EDIT: fixed.

File: tasmota/language/en-GB.h (hunk 1 above)
Comment (ascillato2) [reply tasmota/language/en-GB.h]: Fixed

**Review by ascillato2 — APPROVED** [review]

**Comment by kugelkopf123** [comment]
I don't want to be a spoiler, but as far as I can see this whole translation thing is no longer necessary. The translation only applies if the hard code counter is implemented. This approach has not been developed since the xdrv_10_scripter.
All settings including the naming of the values are now made in the scripter.
So actually that's a lot of unnecessary code. Maybe @gemu2015 can say something about it.

**Comment by Jason2866** [comment]
If you have one of these ```#define EHZ161_0 1
#define EHZ161_1 2
#define EHZ363 3
#define EHZH 4
#define EDL300 5
#define Q3B 6
#define COMBO3 7
#define COMBO2 8
#define COMBO3a 9
#define Q3B_V1 10
#define EHZ363_2 11
#define COMBO3b 12
#define WGS_COMBO 13
#define EBZD_G 14```
and you do not want to use scripter this driver does still his job.

**Comment by kugelkopf123** [comment]
Sure it does!
But wouldn't it save some memory/Flash to remove everything like this from the code and just let it run through the scripter? Apart from that, the standard code that appears after activation of this Driver eg. (OBIS) always leads to confusion.

**Comment by Jason2866** [comment]
@kugelkopf123 
No it does save nothing dropping this driver since `USE_SML_M` is not enabled in any
Tasmota precompiled builds.

So only users which do there own build(s) can activate this driver, i am sure the know what the do.
No reason to drop a working driver that gives a different oppertunity to use.

**Comment by kugelkopf123** [comment]
> @kugelkopf123
> No it does save nothing dropping this driver since `USE_SML_M` is not enabled in any
> Tasmota precompiled builds.
> 
Okay!

> So only users which do there own build(s) can activate this driver, i am sure the know what the do.
Thats funny! I would suggest that you look into the threads over this driver. It is shurly 
> No reason to drop a working driver that gives a different oppertunity to use.


That is a very amusing statement.
I would suggest you check out the forums dealing with this driver. It's really a fantastic driver, no question about it. But to say that the people who activate it know what they're doing is going too far.
I'm pretty active in the SML community and I know what I'm talking about. For this reason I have already written a complete tutorial for it and also started to revise the tasmota wiki.

It was just a suggestion to remove this complete hardcoded meter option. I assumed that it would probably save space and maybe some things in the configuration would be easier to understand for beginners.

**Pull request merged**